content/academy/node_js/apify_free_google_serp_api.md

```diff
@@ -8,8 +8,12 @@ paths:
 
 So you need to regularly grab SERP data about your target keywords? Apify provides a free SERP API that includes organic search, ads, people also ask, etc. Free Apify accounts come with 30 days of proxy trial and $5 of credit. To get started, head over to the [Google Search Results Scraper](https://apify.com/apify/google-search-scraper) page and click the `Try me` button. You'll be taken to a page where you can enter the search query, region, language and other settings.
 
+<<<<<<< HEAD
```

Comment:
these are marking the conflict you need to resolve, you dont wanna commit them.

Reply:
Yeah, I'm fixing them. I will push the fix in a soon.